### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Yash-Garg**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Changes proposed in this pull request:

```
> Some grammatical errors fixed
> Added a link to the Wiki (just in case)
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The **Open Event Server** exposes a well documented [JSON:API Spec](http://jsona
 * Please join our **[mailing list](https://groups.google.com/forum/#!forum/open-event)** to discuss questions regarding the project.
 > https://groups.google.com/forum/#!forum/open-event
 
-* Our chat channel is on **[Gitter](https://gitter.im/fossasia/open-event-server)**
+* Our chat channel on **[Gitter](https://gitter.im/fossasia/open-event-server)**
```
Comment (mayank8318) [inline README.md]: Keep the `is`

File: README.md [hunk 2]
```diff
@@ -278,6 +275,8 @@ The tentative release policy for now is, (since there is a lot of activity and a
 * Test the application on your machine and explore the admin area. The test deployment on Heroku will not give you access to the admin section, where you can switch on/off modules, e.g. ticketing and add keys for services, e.g. storage on S3.
 * If you would like to work on an issue, drop in a comment at the issue. If it is already assigned to someone, but there is no sign of any work being done, please free to drop in a comment so that the issue can be assigned to you if the previous assignee has dropped it entirely.
 
+For more instructions head to the [Wiki](https://github.com/fossasia/open-event-server/wiki).
```
Comment (mayank8318) [inline README.md]: For more information head `over` to the ...

File: README.md (hunk 1 above)
Comment (Yash-Garg) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
 It offers features for events with several tracks and venues. Event managers can create invitation forms for speakers and build schedules in a *drag and drop* interface. The event information is stored in a **database**. The system provides **API endpoints** to **fetch** the data, and to **modify** and **update** it. Organizers can import and export event data in a standard compressed file format that includes the event data in **JSON and binary** media files like **images and audio**.
 
-The **Open Event Server** exposes a well documented [JSON:API Spec](http://jsonapi.org/) Compliant `REST API` that can be used by external services *(like the Open Event App generators and the frontend)* to access & manipulate the data.
+The **Open Event Server** exposes a well documented [JSON: API Spec](http://jsonapi.org/) Compliant `REST API` that can be used by external services *(like the Open Event App generators and the frontend)* to access & manipulate the data as per their choice.
```
Comment (niranjan94) [inline README.md]: Not needed. Please revert this change.

File: README.md [hunk 4]
```diff
@@ -47,12 +47,10 @@ The Open Event Server can be easily deployed on a variety of platforms. Detailed
 1. [Deployment with Docker](/docs/installation/docker.md)
 1. [Deployment on Heroku](/docs/installation/heroku.md)
 
-
-One-click Heroku and Docker deployments are also available:
+One-click Heroku and Docker deployments are also given below:
```
Comment (niranjan94) [inline README.md]: Not needed. Please revert this change.

**Comment by Yash-Garg** [comment]
@niranjan94 Made changes you requested

File: README.md [hunk 5]
```diff
@@ -13,7 +13,7 @@
 
 It offers features for events with several tracks and venues. Event managers can create invitation forms for speakers and build schedules in a *drag and drop* interface. The event information is stored in a **database**. The system provides **API endpoints** to **fetch** the data, and to **modify** and **update** it. Organizers can import and export event data in a standard compressed file format that includes the event data in **JSON and binary** media files like **images and audio**.
 
-The **Open Event Server** exposes a well documented [JSON:API Spec](http://jsonapi.org/) Compliant `REST API` that can be used by external services *(like the Open Event App generators and the frontend)* to access & manipulate the data.
+The **Open Event Server** exposes a well documented [JSON: API Spec](http://jsonapi.org/) Compliant `REST API` that can be used by external services *(like the Open Event App generators and the frontend)* to access & manipulate the data.
```
Comment (iamareebjamal) [inline README.md]: This is unneeded change. `JSON:API` is correct

http://jsonapi.org/

File: README.md [hunk 6]
```diff
@@ -47,12 +47,10 @@ The Open Event Server can be easily deployed on a variety of platforms. Detailed
 1. [Deployment with Docker](/docs/installation/docker.md)
 1. [Deployment on Heroku](/docs/installation/heroku.md)
 
-
```
Comment (iamareebjamal) [inline README.md]: Please re-add the newline

File: README.md [hunk 7]
```diff
@@ -47,12 +47,10 @@ The Open Event Server can be easily deployed on a variety of platforms. Detailed
...
 One-click Heroku and Docker deployments are also available:
 
 [![Deploy to Docker Cloud](https://files.cloud.docker.com/images/deploy-to-dockercloud.svg)](https://cloud.docker.com/stack/deploy/?repo=https://github.com/fossasia/open-event-server)  [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
 
-
```
Comment (iamareebjamal) [inline README.md]: Please re-add the newline

File: README.md (hunk 5 above)
Comment (Yash-Garg) [reply README.md]: ok

File: README.md (hunk 6 above)
Comment (Yash-Garg) [reply README.md]: y tho?

File: README.md (hunk 7 above)
Comment (Yash-Garg) [reply README.md]: why?
